parsec/core/backend_connection/anonymous.py

```diff
@@ -36,11 +41,11 @@
 
 
 async def _anonymous_cmd(
-    serializer: CmdSerializer,
+    serializer: ApiCommandSerializer | CmdSerializer,
     addr: BackendPkiEnrollmentAddr | BackendOrganizationBootstrapAddr,
     organization_id: OrganizationID,
     **req: object,
-) -> dict[str, object]:
+) -> Any:
```

Comment:
Can't this be better typed ? we don't have a enum/ generic type for anonymous cmd ?

Reply:
Can we use an union of all the `Rep` types? I don't see a best fit here.